packages/app/src/app/overmind/namespaces/editor/actions.ts

```diff
@@ -298,9 +300,7 @@ export const sandboxChanged: AsyncAction<{ id: string }> = withLoadApp<{
     actions.comments.getSandboxComments(sandbox.id);
   }
 
-  if (sandbox.originalGit && hasPermission(sandbox.authorization, 'owner')) {
-    actions.git.loadGitSource();
-  }
+  actions.git.loadGitSource();
```

Comment:
Should there still be a check on `originalGit`? Or does this also work without an `originalGit`?

Reply:
I have moved the check inside `loadGitSource`:-)